tests/storages_tests/test_storages.py

```diff
@@ -525,6 +595,30 @@ def test_set_trial_param(storage_mode: str) -> None:
             storage.set_trial_param(non_existent_trial_id, "x", 0.1, distribution_x)
 
 
+def _test_set_and_get_trial_param_for_floats(storage: BaseStorage) -> None:
+    study_id = storage.create_new_study(directions=[StudyDirection.MINIMIZE])
+    trial_id = storage.create_new_trial(study_id)
+
+    for key, value in FLOAT_ATTRS.items():
+        float_distribution = FloatDistribution(low=value, high=value)
+        categorical_distribution = CategoricalDistribution(choices=(value,))
+        for distribution in (float_distribution, categorical_distribution):
+            # NOTE: suggest_float does not generate infinite values.
```

Comment:
Minor comment: Since `math.isfinite` also checks for NaN values, the following comment might be more accurate:
```suggestion
            # NOTE: suggest_float does not generate infinite values or NaN.
```